Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -125,23 +176,24 @@ public override void OnPreSceneQuery()
                 }
 
                 Vector3 pointerPosition;
-                if (TryGetNearGraspPoint(out pointerPosition))
+                Vector3 pointerAxis;
```

Comment:
might want to assert / check that pointerAxis magnitude is 1. I don't think it's mentioned anywhere in the docs that its expected magnitude is 1 either, would be good to mention that in the docs.